src/rapidcsv.h

```diff
@@ -148,13 +148,13 @@ namespace rapidcsv
       else if (typeid(T) == typeid(float))
       {
         std::ostringstream out;
-        out << std::defaultfloat << std::setprecision(9) << pVal;
+        out << std::setprecision(9) << pVal;
```

Comment:
Removing `<< std::defaultfloat` should be harmless since the stream has been created in its default state just above.